src/main/java/seedu/address/github/GithubApi.java

```diff
@@ -54,15 +53,9 @@ public HttpResponse<?> onFail(Exception e, HttpRequestSummary request, Config co
 
     public User getUser(String username) throws UserInvalidException, NetworkConnectionException {
         requireAllNonNull(username);
-        checkUserExists(username, unirest);
         UserInfoWrapper userInfoWrapper = new UserInfoWrapper(username, unirest);
         UserReposWrapper userReposWrapper = new UserReposWrapper(username, unirest);
 
         return new User(username, userInfoWrapper, userReposWrapper);
     }
-
-    private void checkUserExists(String username, UnirestInstance unirest)
```

Comment:
This is done to reduce calls to GitHub API as normal limit is 60/hour. This check will inherently be done anyway when getting user data/repos.